ui/src/main.tsx

```diff
@@ -341,7 +341,10 @@ if (isOnDevice) {
               loader: DeviceIdRename.loader,
               action: DeviceIdRename.action,
             },
-            { path: "devices", element: <DevicesRoute />, loader: DeviceListLoader },
+            { 
+              path: "devices", 
+              element: <DevicesRoute />,
+              loader: DevicesRoute.loader },
```

Comment:
Fixing the eslint warning about the `export const loader` requires a change here to follow the same pattern as other routes.